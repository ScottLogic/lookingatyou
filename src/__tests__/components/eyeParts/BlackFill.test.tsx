import { shallow } from 'enzyme';
import React from 'react';
import {
    BlackFill,
    IBlackFillProps,
} from '../../../components/eye/eyeParts/BlackFill';

let props: IBlackFillProps;

describe('Blackfill', () => {
    beforeEach(() => {
        props = {
            leftX: 20,
            scleraRadius: 300,
            width: 1920,
            height: 1080,
        };
    });

    it('should render correctly', () => {
        const wrapper = shallow(<BlackFill {...props} />).debug();
        expect(wrapper).toMatchSnapshot();
    });
});
