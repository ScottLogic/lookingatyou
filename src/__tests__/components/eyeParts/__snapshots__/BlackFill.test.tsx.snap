// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blackfill should render correctly 1`] = `
"<svg className=\\"BlackFill\\">
  <path d=\\"M 0 540,\\\\n                         H 20,\\\\n                         A 300 300 0 1 1 20 541\\\\n                         v -1\\\\n                         H 0\\\\n                         V 0\\\\n                         H 1920\\\\n                         V 1080\\\\n                         H 0\\\\n                         Z\\" />
</svg>"
`;
